README.md

```diff
@@ -593,6 +593,20 @@ This is the `forge:` key in r10k, it accepts a hash that contains settings for d
     }
 ```
 
+##### `deploy_settings`
+This is the `deploy:` key in r10k, it accepts a hash that contains setting that control how r10k code deployments behave.
+
+```puppet
+    $deploy_settings = {
+      'purge_levels' => ['puppetfile'],
+    }
+
+    class {'r10k':
+      remote          => 'git@github.com:acidprime/puppet.git',
```

Comment:
I prefer to use example references here. Maybe use voxpupuli as the author instead.